Solutions/Nasuni/Data Connectors/Nasuni Data Connector.json

```diff
@@ -0,0 +1,117 @@
+{
+    "id": "NasuniEdgeAppliance",
+    "title": "Nasuni Edge Appliance",
+    "publisher": "Nasuni",
+    "descriptionMarkdown": "The [Nasuni](https://www.nasuni.com/) connector allows you to easily connect your Nasuni Edge Appliance Notifications and file system audit logs with Microsoft Sentinel. This gives you more insight into activity within your Nasuni infrastructure and improves your security operation capabilities.",
+    "graphQueries": [
```

Comment:
Please add a text parameter as "additionalRequirementBanner": "None ",   below descriptrionmarkdown.
As this data connector does not depends on a parser so it can be kept as none.

You can refer to this data connector for better understanding
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Morphisec/Data%20Connectors/Morphisec.json